.openpublishing.publish.config.json

```diff
@@ -48,11 +48,8 @@
     {
       "path_to_root": "_csharplang",
       "url": "https://github.com/dotnet/csharplang",
-      "branch": "master",
-      "include_in_build": true,
-      "branch_mapping": {
```

Comment:
For the vblang repo, we didn't delete the branch_mapping key. We should probably make these two consistent to avoid later confusion.